README.md

```diff
@@ -41,7 +41,7 @@ repo_path = os.path.dirname(os.path.realpath(__file__))
 rpk = os.path.join(repo_path, 'tests/data/extrusion_rule.rpk')
 shape_attributes = {'shapeName': 'myShape', 'seed': 555}
 encoder = 'com.esri.pyprt.PyEncoder'
-encoder_options = {'emitReport': True, 'emitGeometry': True}
+encoder_options = {'emitReport': True, 'emitGeometry': True, 'triangulate': False}
```

Comment:
As this is a _minimal_ example and triangulation is off by default, I would not include it here.